data/NominationsData.old.js

```diff
@@ -1,363 +0,0 @@
-import Aks_Image from "public/img/exec/2022/Aks.jpg";
```

Comment:
Don't want to keep backups of nominations?

Reply:
I guess it's all in git if you really wanted them

Reply:
Yeah, we have the old one and nobody actually needs to know the old list. Entrusting this to Git.